jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/JibExtension.java

```diff
@@ -118,6 +120,15 @@ public JibExtension(Project project) {
         objectFactory.newInstance(ExtensionParametersSpec.class, pluginExtensions);
     allowInsecureRegistries = objectFactory.property(Boolean.class);
     containerizingMode = objectFactory.property(String.class);
+    configurationName =
+        objectFactory
+            .property(String.class)
+            .convention(
+                project.provider(
+                    () ->
+                        System.getProperty(
+                            PropertyNames.CONFIGURATION_NAME,
+                            JavaPlugin.RUNTIME_CLASSPATH_CONFIGURATION_NAME)));
```

Comment:
Need to revert this change: 
```suggestion
    configurationName =
        objectFactory
            .property(String.class)
            .convention(JavaPlugin.RUNTIME_CLASSPATH_CONFIGURATION_NAME);
```